src/components/sidebar/appSelector.tsx

```diff
@@ -316,9 +312,11 @@ export const SidebarAppSelector: React.FC<SidebarAppSelectorProps> = ({
   const appLinks = appsWithoutCurrent.map((app) => {
     const handleClick = () => {
       setOpen(false);
+
       setTimeout(() => {
-        app.onClick();
+        if (app.onClick) app.onClick();
```

Comment:
Is this meant as a null/undefined check? Can't I use `app?.onClick()` in Typescript? If not, something I have to keep in mind for the pulumi stuff also.

Reply:
@dereulenspiegel Yes. I didn't know it was possible before with methods, but I just checked and this is the syntax for optional changing a method and calling it: `app.onClick?.();`. I will fix this quickly and keep it in mind for the future.